org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler</id>
```

Comment:
changing the ID should also be in the manifest and the files should be renamed too to match.